Fill out Guide for Internationalization (#38293)
mdn/content: files/en-us/web/javascript/reference/global_objects/intl/listformat/listformat/index.md

```js
const list = ["Motorcycle", "Bus", "Car"];
console.log(new Intl.ListFormat("en-US", { style: "short" }).format(list));
// Motorcycle, Bus, & Car

console.log(new Intl.ListFormat("en-US", { style: "narrow" }).format(list));
// Motorcycle, Bus, Car

console.log(new Intl.ListFormat("en-GB", { style: "short" }).format(list));
// Motorcycle, Bus and Car

console.log(new Intl.ListFormat("en-GB", { style: "narrow" }).format(list));
// Motorcycle, Bus, Car
```

The input elements are not transformed, but you often want to abbreviate them too.

```js
const marathon = [
  [42, "kilometer"],
  [195, "meter"],
];

function formatDistance(locale, distance, style) {
  return new Intl.ListFormat(locale, { type: "unit", style }).format(
    marathon.map((component) =>
      component[0].toLocaleString(locale, {
        style: "unit",
        unit: component[1],
        unitDisplay: style,
      }),
    ),
  );
}

console.log(formatDistance("en-US", marathon, "long"));
// 42 kilometers, 195 meters
console.log(formatDistance("en-US", marathon, "short"));
// 42 km, 195 m
console.log(formatDistance("en-US", marathon, "narrow"));
// 42km 195m
```

### Selection of conjunction

> The choice of the conjunction may be dependent on the string value of the list items. For example, in Spanish, the conjunction is `"y"` for most words, but `"e"` for words starting with the vowel `"i"`.

Author: I find it ungrammatical to not include "the", because you are choosing between one of several conjunctions (a conjunction is a type of word). I also slightly prefer "X of Y" to "Y's X".

> The choice of the conjunction may be dependent on the string value of the list items. For example, in Spanish, the conjunction is `"y"` for most words, but `"e"` for words starting with the vowel `"i"`.

Reviewer: Your point is reasonable, but "The choice of the..." sounds really clunky.

Even better would be "The conjugation choice...", which would declunkify it and solve your grammar concern.

> The choice of the conjunction may be dependent on the string value of the list items. For example, in Spanish, the conjunction is `"y"` for most words, but `"e"` for words starting with the vowel `"i"`.

Author: ```suggestion
The conjunction word used may depend on the list items' string values. For example, in Spanish, the conjunction is `"y"` for most words, but `"e"` for words starting with the vowel `"i"`.
```
